packages/ui/src/utils/env.ts

```diff
@@ -1 +1,5 @@
-export const __DEV__ = typeof process !== 'undefined' && process.env.NODE_ENV !== 'production'
+const processShim: typeof process = typeof process !== 'undefined' ? process : ({} as typeof process)
+const envShim = processShim.env || ({} as typeof process.env)
+const nodeEnv = envShim.NODE_ENV || ''
```

Comment:
```suggestion
const nodeEnv = envShim.NODE_ENV || 'prod'
```

Let's make it prod by default if there are not process

Reply:
This will break warnings in demos, cause there're no process. Have made a separate issue for this: https://github.com/epicmaxco/vuestic-ui/issues/3141